README.md

```diff
@@ -104,6 +104,12 @@ First, install ocagent if you haven't.
 $ go get github.com/census-instrumentation/opencensus-service/cmd/ocagent
 ```
 
+Alternatively you can build ocagent from binary (from the root of your repo):
+
+```shell
+$ make agent
```

Comment:
So even if they use `go get` without `GO111MODULE=on` it'll trip out unfortunately.